src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/ReviewUnusedParameters.Fixer.cs

```diff
@@ -23,8 +31,144 @@ public sealed override FixAllProvider GetFixAllProvider()
         public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
         {
             // Fixer not yet implemented.
+            Diagnostic diagnostic = context.Diagnostics.Single();
+
+            context.RegisterCodeFix(
+                new MyCodeAction(
+                    MicrosoftMaintainabilityAnalyzersResources.ReviewUnusedParametersMessage,
+                    async ct => await RemoveNodes(context.Document, diagnostic, ct).ConfigureAwait(false), diagnostic.Id),
```

Comment:
> diagnostic.Id

Instead of using diagnostic id as the equivalence key, please use the new resource string that you define for the code fix title.